### PR TITLE
BRANCH-CLIFF-changes

### PR DESCRIPTION
**Pull request by gitcliff**
these are the changes made for this new module repository called patientcolors

### CODE REVIEW

File: omod/src/main/java/org/openmrs/module/patientcolors/web/controller/PatientcolorsController.java [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+import org.springframework.validation.BindingResult;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+
+/**
+ * This class configured as controller using annotation and mapped with the URL of
+ * 'module/${rootArtifactid}/${rootArtifactid}Link.form'.
+ */
+@Controller("${rootrootArtifactid}.PatientcolorsController")
```
Comment (mozzy11) [inline omod/src/main/java/org/openmrs/module/patientcolors/web/controller/PatientcolorsController.java]: change all the occurrences of `${rootrootArtifactid}` to `patientcolors`

Comment (gitcliff) [reply omod/src/main/java/org/openmrs/module/patientcolors/web/controller/PatientcolorsController.java]: thanks man gonna work on it

Comment (mozzy11) [reply omod/src/main/java/org/openmrs/module/patientcolors/web/controller/PatientcolorsController.java]: actually first try correcting `${rootrootArtifactid}` to` ${rootArtifactid}`. only here , and build the module again and install it , and see if its running

Comment (mozzy11) [reply omod/src/main/java/org/openmrs/module/patientcolors/web/controller/PatientcolorsController.java]: first wait to implement this  

>  change all the occurrences of ${rootrootArtifactid} to patientcolors

**Pull request merged**